### DISCUSSION TITLE
Таблица циклов не загружается

### DISCUSSION THREAD

**Discussion by tvideo** [totumonline/totum-ru-issues-and-discussions#457] [ОШБК!] [answered]
Приветствую. Помогите с одним вопросиком. Есть таблица циклов, в циклах 3 таблицы Расчетные в цикле. В главную таблицу циклов были выведены все данные из расчетных в цикле таблиц. Все работало, там уже рабочие данные. Я добавил одно поле (Текстовый) в колонку,  при добавлении поля крутилось шестерня бесконечно, я обновил страницу и теперь таблица циклов не загружается (очень долго грузится и так и не загружается), а там рабочие данные уже. Через список таблиц вижу эту таблицу циклов, но в ее составе нет того текстового поля в колонке, которое я добавлял, думал его удалить и это решит проблему.
В чем может быть причина? 
спасибо

**Comment by tvideo** [comment]
Добавил в простой таблице обычное поле, и с этой таблицей случилось то, что и с таблицей циклов. В чем может быть причина? 
Спасибо

**Reply by ttm-alexeypolunin** [reply, marked as answer]
Форум вам вряд-ли поможет.
